codebuild/spec/buildspec_ubuntu.yml

```diff
@@ -42,7 +42,7 @@ phases:
         if expr "${LATEST_CLANG}" != "true" >/dev/null; then
           apt-get install -y --no-install-recommends clang-3.9 llvm-3.9;
         fi
-      - apt-get install -y --no-install-recommends indent kwstyle lcov libssl-dev m4 make net-tools nettle-bin nettle-dev pkg-config psmisc python3-pip shellcheck sudo tcpdump unzip valgrind zlib1g-dev zlibc cmake tox libtool ninja-build
+      - apt-get install -y --no-install-recommends indent kwstyle lcov libssl-dev m4 make net-tools nettle-bin nettle-dev pkg-config psmisc python3-pip shellcheck sudo tcpdump unzip valgrind zlib1g-dev zlibc cmake tox libtool ninja-build quilt
```

Comment:
Same comments as below in the `*_awslc.yml` file. I'm not sure whether these two .yml files interact with each other?

Reply:
No interaction.